testsuite/MDAnalysisTests/utils/test_distances.py

```diff
@@ -325,7 +355,7 @@ def test_selfdist(self):
 
     def test_distarray(self):
         from MDAnalysis.lib.distances import distance_array
-        from MDAnalysis.lib.distances import transform_StoR, transform_RtoS
+        from MDAnalysis.lib.distances import transform_StoR
```

Comment:
these modules should already be imported anyway. Do the tests fail when you remove the imports? Just keep in mind that they are in the `MDAnalysis.lib.distances` namespace globally in that file.